.github/workflows/ci_testing.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
         run: ansible-galaxy collection install containers.podman -f
 
       - name: Docker login
-        run: docker login -u="redhat_cop+github_action" -p="${{ secrets.quay_token }}" quay.io
+        run: docker login -u="redhat_cop+gith_action" -p=${{ secrets.QUAY_TOKEN }} quay.io
```

Comment:
Is this correct? the CI is failing

Reply:
Its not, I was trying to fix it as was failing before, I've fixed the user name, but expect it to still fail.